### PR TITLE
DOCU-521: Added constant URL for az-cli python wheel.

### PR DESCRIPTION
**Pull request by dafunker**
* "Latest" address should remain constant
    and developers will deploy updates there
    so as to avaoid changes to the doc'd URL.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,5 @@ website/build/
 website/yarn.lock
 website/node_modules
 website/i18n/*
+
+*.iml
```
Comment (benradstone) [inline .gitignore:14]: @dafunker Hey Gordon, just wondering why this change was needed?

Comment (dafunker) [reply .gitignore:14]: I added the project to my Intellij IDE. It creates an iml file, so simply wanted to ignore that. Might help if you'd like other developers to contribute to the document space.

Comment (benradstone) [reply .gitignore:14]: Thanks for clarifying!
